calour/heatmap/heatmap.py

```diff
@@ -164,8 +164,11 @@ def heatmap(exp: Experiment, sample_field=None, feature_field=None,
     ----------
     sample_field : str or None, optional
         The field of sample metadata to display on the x-axis or None (default) to not show x axis.
-    feature_field : str or None, optional
-        The field of feature meadata to display on the y-axis or None (default) to not show y axis.
+    feature_field : str or None or False (optional)
+        The field of feature meadata to display on the y-axis.
+        False to use the default feature field (set by experiment type)
+        None (default) to not show y axis.
+    xticklabel_kwargs, yticklabel_kwargs : dict or None (optional)
```

Comment:
pls don't change the above line. the price we pay to drop numpydoc is that we can have multiple func parameters on the same line - that's why I changed in my last PR

Reply:
fixed.
byebye numpydoc :)